mlflow/pipelines/step.py

```diff
@@ -115,6 +115,7 @@ def run(self, output_directory: str):
         self._initialize_databricks_spark_connection_and_hooks_if_applicable()
         try:
             self._update_status(status=StepStatus.RUNNING, output_directory=output_directory)
+            self._init_from_pipeline_config()
```

Comment:
In its current context, it's probably better called `_init_from_step_config`. Sorry for the back-and-forth change.